### PR TITLE
[Enhance] Allow using custom fps or input fps in video demo

### PR DESCRIPTION
**Pull request by ckkelvinchan**
## Motivation
Users can set the fps of the output video, or use the fps of the input video.

## Modification
Modify  
1. demo/restoration_video_demo.py
2.  mmedit/apis/restoration_video_inference.py

### CODE REVIEW

File: demo/restoration_video_demo.py [hunk 1]
```diff
@@ -57,15 +62,23 @@ def main():
...
-                                         args.window_size, args.start_idx,
-                                         args.filename_tmpl, args.max_seq_len)
+    output, fps = restoration_video_inference(model, args.input_dir,
+                                              args.window_size, args.start_idx,
+                                              args.filename_tmpl,
+                                              args.max_seq_len)
 
     file_extension = os.path.splitext(args.output_dir)[1]
+    if args.fps is None:
+        if fps is None and file_extension in VIDEO_EXTENSIONS:
```
Comment (Yshuo-Li) [inline demo/restoration_video_demo.py:72]: The logic is different with 
'If the input is not a video, you must specify the frame rate of the output video.'

Comment (ckkelvinchan) [reply demo/restoration_video_demo.py:72]: "args.fps" is None means the user did not specify the frame rate.
"fps is None" means the input is not a video.
"file_extension in VIDEO_EXTENSIONS" means the output is a video.

There will be an error when the input is not a video and the user did not specify the frame rate of the output video. I guess it should be ok?

Comment (Yshuo-Li) [reply demo/restoration_video_demo.py:72]: OK, I see.

**Review by Yshuo-Li — APPROVED** [review]

**Comment by wangruohui** [comment]
Should we continue to merge this?

**Comment by ckkelvinchan** [comment]
@wangruohui Let me double check.

**Comment by ckkelvinchan** [comment]
@wangruohui ok to merge~

**Comment by zengyh1900** [comment]
This pull request is closed because it has been open for 30 days with no activity.

**Pull request closed without merging**